frontend/src/metabase-lib/parameters/utils/targets.ts

```diff
@@ -56,3 +56,20 @@ export function getTargetFieldFromCard(
   const field = getParameterTargetField(target, metadata, question);
   return field ?? null;
 }
+
+export function compareMappingOptionTargets(
```

Comment:
Since it's only used in `auto-wire-parameters/actions.ts` and `auto-wire-parameters/utils.ts`, how about we just move it to `auto-wire-parameters/utils.ts` to keep it local to that directory?

Reply:
I'll page @kulyk for this one 😄

Reply:
Let's leave it in `metabase-lib` — it should consolidate complex business logic for queries, parameters, etc.